zomes/main/code/Cargo.toml

```diff
@@ -8,9 +8,9 @@ edition = "2018"
 serde = "=1.0.89"
 serde_json = { version = "=1.0.39", features = ["preserve_order"] }
 serde_derive = "=1.0.89"
-hdk = { git = "https://github.com/holochain/holochain-rust", tag = "v0.0.31-alpha1" }
-hdk-proc-macros = { git = "https://github.com/holochain/holochain-rust", tag = "v0.0.31-alpha1" }
-holochain_wasm_utils = { git = "https://github.com/holochain/holochain-rust", tag = "v0.0.31-alpha1" }
+hdk = "0.0.38-alpha14"
+hdk_proc_macros = "0.0.38-alpha14"
+holochain_wasm_utils = "0.0.38-alpha14"
```

Comment:
This is now re-exported by the hdk crate so this could be removed and the `extern crate` replaced with `use hdk::holochain_wasm_utils`